ATS/db/seeds.rb

```diff
@@ -35,90 +35,112 @@
 
 College.create(colleges)
 
-departments = [{name:  "Computer Science", college_id: 1},
-	{name:  "Criminal Justice/Legal Studies", college_id: 1},
-	{name:  "English", college_id: 1},
-	{name:  "Humanities", college_id: 1},
-	{name:  "Mathematics", college_id: 1},
-	{name:  "Music", college_id: 1},
-	{name:  "Kinesiology and Exercise Science", college_id: 1},
-	{name:  "Natural and Physical Sciences", college_id: 1},
-	{name:  "Social and Behavioral Sciences", college_id: 1},
-	{name:  "Bibilical Studies", college_id: 2},
-	{name:  "Vocational Christian Ministry", college_id: 2},
-	{name:  "Business", college_id: 3},
-	{name:  "Education", college_id: 4},
+AS = College.find_by_name("College of Arts and Sciences")
+Bib = College.find_by_name("College of Bible")
+departments = [{name:  "Computer Science", college_id: AS},
```

Comment:
@sah0017 It appears you are trying to pass the entire object into the `college_id` field. You should be able to write `AS.id` to remedy this. Activerecord will not recognize the Activerecord object and automatically find the id if it is passed through a hash, instead you have to set college_id (and in other places degree_type and department_id) to the actual integer.